src/views/List.jsx

```diff
@@ -53,42 +53,65 @@ export function List({ data, listToken, isLoading }) {
 		}
 	};
 	return (
-		<>
+		<div className="min-h-[100vh]">
```

Comment:
This `min-height` here causes a vertical scroll even if the content is less than the screen size.